sklearn/tree/_tree.pyx

```diff
@@ -695,13 +695,13 @@ cdef class Tree:
     node_count : int
         The number of nodes (internal nodes + leaves) in the tree.
 
+    max_depth : int
```

Comment:
Can be placed in it's original position to reduce the diff with `main`?

Reply:
Done